cmake/singularity/kokkos.cmake

```diff
@@ -0,0 +1,112 @@
+macro(singularity_import_kokkos)
+  
+  set(Kokkos_ENABLE_SERIAL ON CACHE BOOL "" FORCE)
+  if(SINGULARITY_USE_CUDA)
+    set(Kokkos_ENABLE_CUDA ON CACHE BOOL "" FORCE)
+    set(Kokkos_ENABLE_CUDA_LAMBDA ON CACHE BOOL "" FORCE)
+    set(Kokkos_ENABLE_CUDA_RELOCATABLE_DEVICE_CODE ON CACHE BOOL "" FORCE)
```

Comment:
We should definitely remove this. We no longer require rdc.

Reply:
Clarification: remove _just_ `RELOCATABLE`, or remove all but `Kokkos_ENABLE_CUDA`, or remove all these cache options entirely?

Reply:
done